Content.Server/Gravity/EntitySystems/GravitySystem.cs

```diff
@@ -48,16 +48,17 @@ public override void Update(float frameTime)
 
             foreach (var grid in _mapManager.GetAllGrids())
             {
-                if (grid.HasGravity && !gridsWithGravity.Contains(grid.Index))
+                var gridEntity = EntityManager.GetEntity(grid.GridEntityId);
```

Comment:
I'm not sure if this is the correct pattern for checking components existence on Entities